python/desc/sims/GCRCatSimInterface/ProtoDC2DatabaseEmulator.py

```diff
@@ -228,6 +229,24 @@ def _transform_catalog(self, gc):
         gc.add_quantity_modifier('sindex::disk', gc.get_quantity_modifier('sersic_disk'))
         gc.add_quantity_modifier('sindex::bulge', gc.get_quantity_modifier('sersic_bulge'))
 
+        # Test for random walk specific addon
```

Comment:
Can you put this block of code into a `_transform_knots()` method defined in `DESCQAObject`, and then have both `DESCAObject._transform_catalog()` and `DESCQAObject_protoDC2._transform_catalog()` call `_transform_knots()`?  That way, we only have to worry about changing `_transform_knots()` if we end up having to develop on this code.

Reply:
very good suggestion, the code had already diverged between the two `_transform_catalog()` functions :-/ . Done